### PR TITLE
Enhance centcore log

### PR DESCRIPTION
**Pull request by garnier-quentin**
<h1> Pull Request Template </h1>

<h2> Description </h2>

Enhance cetncore log messages.

<h2> Type of change </h2>

Please delete options that are not relevant.

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

<h2> Target serie </h2>

Please delete series that are not relevant.

- [x] 18.10.x
- [x] 19.04.x (master)

<h2> How this pull request can be tested ? </h2>

Look message in centcore log message. You should find 'INFO', 'DEBUG' or 'ERROR'.

<h2> Checklist </h2>

<h5> Community contributors & Centreon team </h5>

- [x] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).
- [ ] I have updated the **release note** in dedicated temporary section **\***

<h5> Centreon team only </h5>

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests covers 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: lib/perl/centreon/script/centcore.pm [hunk 1]
```diff
@@ -685,12 +689,16 @@ sub sendConfigFile($){
...
                 $cmd = "cp $origin $cfg_dir 2>&1";
-                ($lerror, $stdout) = centreon::common::misc::backtick(
+                ($lerror, $stdout, $return_code) = centreon::common::misc::backtick(
                     command => $cmd,
                     logger => $self->{logger},
                     timeout => 60
                 );
             }
+                            
+            if ($lerror != 0 || (defined($return_code)  && $return_code != 0)) {
```
Comment (sc979) [inline lib/perl/centreon/script/centcore.pm]: I think there's a space to remove between : `$return_code)` and `&& $return_`

File: lib/perl/centreon/script/centcore.pm [hunk 2]
```diff
@@ -632,13 +632,17 @@ sub sendConfigFile($){
...
     }
 
-    ($lerror, $stdout) = centreon::common::misc::backtick(
+    ($lerror, $stdout, $return_code) = centreon::common::misc::backtick(
         command => $cmd,
         logger => $self->{logger},
         timeout => 300
     );
+    
+    if ($lerror != 0 || (defined($return_code)  && $return_code != 0)) {
```
Comment (sc979) [inline lib/perl/centreon/script/centcore.pm]: same as line 699

File: lib/perl/centreon/script/centcore.pm (hunk 1 above)
Comment (garnier-quentin) [reply lib/perl/centreon/script/centcore.pm]: For me it's ok. We should merge that kind of stuff in one PR clean code.

**Comment by adr-mo** [comment]
**Merged on master and 18.10.x**

**Pull request closed without merging**